### PR TITLE
Still check for players using the HUD even if no lister ents are spawned

### PR DESCRIPTION
**Pull request by legokidlogan**
Fixes [this unreported issue](https://github.com/edshot99/cfc_chip_lister/commit/28f4a2455520dfcdede22e8d9cb586ca83753ad3)

### CODE REVIEW

File: lua/autorun/server/sv_chip_lister.lua [hunk 1]
```diff
@@ -115,9 +115,11 @@ local function isPlateVisible( ent, ply )
 end
 
 -- Can a player see at least one chip lister?
+-- Give listers as false to only check for the HUD setting (i.e. no lister entities exist).
 local function canSeeALister( ply, listers )
     if not IsValid( ply ) then return false end
     if ply:GetInfoNum( "cfc_chiplister_hud_persist", 0 ) == 1 then return true end
+    if not listers then return end
```
Comment (wrefgtzweve) [inline lua/autorun/server/sv_chip_lister.lua:122]: What's the point of this? you should be returning earlier here, no need to loop over the listUsers only to return empty each time

Comment (legokidlogan) [reply lua/autorun/server/sv_chip_lister.lua]: When no lister ents exist, I still need to figure out who does/doesn't have the HUD enabled, to figure out:
A) Does at least one person have the HUD enabled, i.e. do we bother running an update at all?
B) Who do we need to network the data to?

File: lua/autorun/server/sv_chip_lister.lua [hunk 2]
```diff
@@ -134,7 +136,10 @@ local function getVisibleListUsers()
     if listUserCount == 0 then return {}, 0 end
 
     local listers = ents.FindByClass( "cfc_chip_lister" )
-    if #listers == 0 then return {}, 0 end
+
+    if #listers == 0 then
+        listers = false
```
Comment (wrefgtzweve) [inline lua/autorun/server/sv_chip_lister.lua:141]: `getVisibleListUsers` shouldn't be looping over all the players when no listers are found, this should return. Type mixing should also not be done

Comment (legokidlogan) [reply lua/autorun/server/sv_chip_lister.lua:141]: The whole point of this PR is to make it still loop through the players when no lister is found. Players should not have to spawn a lister entity in order to use the HUD.

**Review by wrefgtzweve — APPROVED** [review]

**Pull request merged**